### ISSUE TITLE
Its app doesn't working in Mac OS (15.1.1)

### ISSUE THREAD

**Issue by toshiki-higa** [toshiki-higa/plux-recorder#1]
Its app doesn't working in Mac OS (MacOS 15.1.1). 

Log is: 
```bash
[BTH MACOS] GOT HERE
[BTH_MACOS] My inport in 15107
path: /Users/toshiki-higa/Documents/code/[github.com/toshiki-higa/plux-recorder/PLUX-API-Python3/M1_312/bth_macprocess](https://github.com/toshiki-higa/plux-recorder/PLUX-API-Python3/M1_312/bth_macprocess)
[BTH_MACOS] Gonna kill all processes for this mac address that might be left over! (kill -9 $(ps aux | grep -w 'bth_macprocess 00:07:80:8C:0A:09' | head -1))
sh: line 0: kill: toshiki-higa: arguments must be process or job IDs
sh: line 0: kill: (98240) - No such process
sh: line 0: kill: 0.0: arguments must be process or job IDs
sh: line 0: kill: 0.0: arguments must be process or job IDs
sh: line 0: kill: (410210752) - No such process
sh: line 0: kill: (1408) - No such process
sh: line 0: kill: s042: arguments must be process or job IDs
sh: line 0: kill: S+: arguments must be process or job IDs
sh: line 0: kill: 4:19PM: arguments must be process or job IDs
sh: line 0: kill: 0:00.01: arguments must be process or job IDs
sh: line 0: kill: grep: arguments must be process or job IDs
sh: line 0: kill: -w: arguments must be process or job IDs
sh: line 0: kill: bth_macprocess: arguments must be process or job IDs
sh: line 0: kill: 00:07:80:8C:0A:09: arguments must be process or job IDs
[BTH_MACOS] Executing is /Users/toshiki-higa/Documents/code/[github.com/toshiki-higa/plux-recorder/PLUX-API-Python3/M1_312/bth_macprocess](https://github.com/toshiki-higa/plux-recorder/PLUX-API-Python3/M1_312/bth_macprocess)
[BTH_MACOS] Spwan Status 0
[MacOS Process] - My Object path is 00:07:80:8C:0A:09
My mac outPort is 2563
My mac inport is 2819
[BTH_MACOS] My outport in 12035
[MacOS Process] - 54, 54
[BTH_MACOS] Configuration passed!
Finished BTH Connection 00:07:80:8C:0A:09
[MacOS Process] - NOT PAIRED
[MacOS Process] - devicePairingStarted!
[MacOS Process] - Pair IORETURN IS 0
[MacOS Process] - devicePairingStarted!
[MacOS Process] - Timeout!
[BTH_MACOS] Timeout from process
[BTH_MACOS] Killing
[BTH_MACOS] Kill aux process with code 0
Exception in thread Thread-26:
Traceback (most recent call last):
  File "/Users/toshiki-higa/.local/share/uv/python/cpython-3.12.0-macos-aarch64-none/lib/python3.12/threading.py", line 1052, in _bootstrap_inner
    self.run()
  File "/Users/toshiki-higa/Documents/code/[github.com/toshiki-higa/plux-recorder/src/app.py](https://github.com/toshiki-higa/plux-recorder/src/app.py)", line 134, in run
    self.device = SignalsDevice(self.mac_address)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: The device could not be found.
```

**Comment by toshiki-higa** [comment]
The problem was solved after updating to MacOS (15.2).

**Comment by toshiki-higa** [comment]
The problem was solved after updating to MacOS (15.2).
